ubuntu-advantage-client/ec2_cleanup.py

```diff
@@ -19,62 +19,125 @@ def parse_args():
         help=("Tag used to filter cloud resources for deletion. "
               "Default: {}".format(CI_DEFAULT_TAG))
     )
+    parser.add_argument(
+        "-o", "--older-than", dest="older_than", action="store",
+        help=("Tag used to filter cloud resources for deletion. "
+              "Format: MM/DD/YY [HH:MM:SS].)"
+    )
     return parser.parse_args()
 
 
-def clean_ec2(tag):
+def get_tag_prefix(older_than):
```

Comment:
I think this needs a docstring about the older_than arg expected data type.

Reply:
+1 done